src/helm/benchmark/static/schema.yaml

```diff
@@ -2954,540 +2984,560 @@ run_groups:
+  - name: cleva_sentiment_analysis
+    display_name: CLEVA (Chinese) sentiment analysis
+    description: Chinese sentiment analysis for product reviews.
     metric_groups:
       - accuracy
       - calibration
+      - efficiency
       - general_information
     environment:
       main_name: exact_match
       main_split: test
     taxonomy:
-      task: "?"
-      what: "?"
-      who: "?"
-      when: "?"
+      task: sentiment analysis
+      what: product reviews
+      who: customers
+      when: 2021 or before
       language: Chinese
 
-  - name: cleva_dialogue_generation
-    display_name: CLEVA (Chinese) dialogue generation
-    description: CLEVA (Chinese) dialogue generation
+# Language
+  - name: cleva_language_modeling
+    display_name: CLEVA (Chinese) language modeling
+    description: Scenario for measuring language model performance across various domains (Wiki and news).
     metric_groups:
       - accuracy
       - calibration
-      - cleva_dialogue_generation_metrics
+      - efficiency
       - general_information
     environment:
-      main_name: chinese_bleu_1
+      main_name: bits_per_byte
       main_split: test
     taxonomy:
-      task: "?"
-      what: "?"
-      who: "?"
-      when: "?"
+      task: language modeling
+      what: Wikipedia and news
+      who: n/a
+      when: 2010s
       language: Chinese
 
-  - name: cleva_fact_checking
-    display_name: CLEVA (Chinese) fact checking
-    description: CLEVA (Chinese) fact checking
+  - name: cleva_pinyin_transliteration
+    display_name: CLEVA (Chinese) pinyin transliteration
+    description: Scenario that asks the model to translate between Chinese and Pinyin.
     metric_groups:
-      - accuracy
-      - calibration
+      - cleva_pinyin_transliteration_metrics
       - general_information
+      - efficiency
     environment:
-      main_name: exact_match
+      main_name: chinese_bleu_1
       main_split: test
     taxonomy:
-      task: "?"
-      what: "?"
-      who: "?"
-      when: "?"
-      language: Chinese
+      task: pinyin transliteration
+      what: n/a
+      who: automatically generated by algorithm
+      when: n/a
+      language: Chinese, Pinyin
 
-  - name: cleva_inductive_reasoning
-    display_name: CLEVA (Chinese) inductive reasoning
-    description: CLEVA (Chinese) inductive reasoning
+  - name: cleva_classical_chinese_understanding
+    display_name: CLEVA (Chinese) classical Chinese understanding
+    description: Scenario for evaluating the understanding of classical Chinese by selecting the appropriate classical Chinese translation for a given modern Chinese sentence.
     metric_groups:
       - accuracy
       - calibration
+      - efficiency
       - general_information
     environment:
       main_name: exact_match
       main_split: test
     taxonomy:
-      task: "?"
-      what: "?"
-      who: "?"
-      when: "?"
-      language: Chinese
+      task: multiple-choice
```

Comment:
I think this is a "paraphrase" or maybe "translation" task.

Reply:
My vote is "translation".